universal-login-react/test/core/services/BackupCodesService.test.ts

```diff
@@ -36,12 +36,13 @@ describe('UNIT: BackupCodesService', () => {
       await backupCodesService.generate(undefined);
       expect(backupCodesService.state.get()).to.deep.eq({
         kind: 'Failure',
-        error: 'Missing gas parameters',
+        error: 'Missing parameter: gas parameters',
       });
     });
 
     it('successful call', async () => {
       await backupCodesService.generate(INITIAL_GAS_PARAMETERS);
+      sleep(1000);
```

Comment:
We should have posibility to waiting for finish generating if `await backupCodesService.generate(INITIAL_GAS_PARAMETERS)` is not enough